site/pages/create-consequence-operator.tsx

```diff
@@ -0,0 +1,209 @@
+                                    cells: [
+                                        previousConsequenceInformation.consequenceType,
+                                        <Link key={"consequence-type"} className="govuk-link" href="/add-consequence">
+                                            Change
+                                        </Link>,
+                                    ],
+                                },
+                            ]}
+                        />
+
+                        <Select<PageInputs>
+                            inputId="consequence-operator"
+                            inputName="consequence-operator"
+                            display="Who is the operator?"
+                            displaySize="l"
+                            defaultDisplay="Select an operator"
+                            errorMessage="Select an operator from the dropdown"
+                            selectValues={OPERATORS}
+                            stateUpdater={stateUpdater}
+                            value={pageState.inputs["consequence-operator"]}
+                        />
+
+                        <TextInput<PageInputs>
+                            inputId="description"
+                            display="Consequence description"
+                            displaySize="l"
+                            inputName="description"
+                            errorMessage="Enter a description for this consequence"
+                            widthClass="w-3/4"
+                            textArea
+                            rows={3}
+                            maxLength={500}
+                            stateUpdater={stateUpdater}
+                            value={pageState.inputs.description}
+                        />
+
+                        <Radios<PageInputs>
+                            display="Would you like to remove this from journey planners?"
+                            displaySize="l"
+                            inputId="remove-from-journey-planners"
+                            radioDetail={[
+                                {
+                                    value: "yes",
+                                    display: "Yes",
+                                },
+                                {
+                                    value: "no",
+                                    display: "No",
+                                },
+                            ]}
+                            inputName="remove-from-journey-planners"
+                            stateUpdater={stateUpdater}
+                            value={pageState.inputs["remove-from-journey-planners"]}
+                        />
+
+                        <TimeSelector<PageInputs>
+                            display="How long is the disruption delay?"
+                            displaySize="l"
+                            hint="Enter the time in 24hr format. For example 0900 is 9am, 1730 is 5:30pm"
```

Comment:
The hint text needs to be slightly different here, as they're providing an amount of time, not a time. Im actually curious if we need a different input here, because they could enter 4800 (for 48 hours) and that would be acceptable. Or 0090 for 90 minutes.

Reply:
done renamed hint